CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Bug fixes
 
+* [#5752](https://github.com/bbatsov/rubocop/pull/5752): Add `String#delete_{prefix,suffix}` to Lint/Void cop. ([@bdewater][])
```

Comment:
This should be written in the `Changes` section instead of the` Bug fixes` section.